File: HASHING/count_frequency.java
import java.util.HashMap;

public class count_frequency {
    public static void main(String[] args) {
        int[] arr = {1, 3, 2, 1, 3, 4, 1};

        HashMap<Integer, Integer> map = new HashMap<>();

        for (int num : arr) {
            map.put(num, map.getOrDefault(num, 0) + 1);
        }

        System.out.println("Element : Frequency");
        for (int key : map.keySet()) {
            System.out.println(key + " : " + map.get(key));
        }
    }
}

File: HASHING/count_frequency.cpp
#include <iostream>
#include <unordered_map>
using namespace std;

int main() {
    int arr[] = {1, 3, 2, 1, 3, 4, 1};
    int n = sizeof(arr) / sizeof(arr[0]);

    unordered_map<int, int> freq;

    for (int i = 0; i < n; ++i) {
        freq[arr[i]]++;
    }

    cout << "Element : Frequency" << endl;
    for (auto &pair : freq) {
        cout << pair.first << " : " << pair.second << endl;
    }

    return 0;
}